src/tools/gophish_cleaner.py

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env python3
 
-"""GoPhish Cleaner for a Phishing Campaign Assessment (PCA).
-
-A tool to remove aspects of an Assessment or a campaign from GoPhish.
+"""Remove an assessment or elements of an assessment.
```

Comment:
I would add `" in GoPhish."` to the end of this first line.  I think it's important to make it clear in the first docopt line if the utility talks to GoPhish (as opposed to simply modifying an assessment JSON file).  

The same goes for the other `gophish-*` scripts.